app/config/datasources.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
since we're probably using always postgres it could make sense to add postgres as a datasource. how are secrets stored?

Reply:
Afaik you should inject them using env variables, all config can be done in js files.